hugegraph-test/src/main/java/com/baidu/hugegraph/core/TaskCoreTest.java

```diff
@@ -429,16 +432,13 @@ public void testGremlinJobWithError() throws TimeoutException {
         });
 
         // Test failure task with big input
-        char[] chars = new char[65536];
-        for (int i = 0; i < chars.length; i++) {
-            chars[i] = '8';
-        }
-        String bigInput = new String(chars);
+        int length = 65536 * 1024;
```

Comment:
add test 128kb is ok

Reply:
128kb < 64MB, will not trigger exception